src/OpenTK/Input/Key.cs

```diff
@@ -121,6 +121,10 @@ public enum Key
         /// </summary>
         Menu,
 
+        /// <summary>The Command key.</summary>
+        /// <remarks>Valid on OS-X only</remarks>
+        Command,
```

Comment:
Argggh, this took me like 4 hours to figure out. 
I tried your patch on my fork. This one enum entry broke all keyboard inputs on mac os for my game because it caused all subsequent enum indices to be shifted by 1 which caused my game internal keycode representation to be incorrectly mapped. 
This is not a bug per se but perhaps its a good idea to move this enum entry to the very bottom for backwards compatibility?

Reply:
Dunno.

I would have thought most people are using the actual enum values, rather than the index directly?
It was deliberately added there so it was grouped with the other OS-X modifiers rather than elsewhere.

You've absolutely proved my point about this biting us though!

Reply:
I have an abstraction layer to abstract away OpenTK itself so I can switch to another graphics library / toolkit should the need ever arise.